--- node_modules/nodecastor/node_modules/protobufjs/ProtoBuf.orig.js	2014-05-13 23:04:53.000000000 +0100
+++ node_modules/nodecastor/node_modules/protobufjs/ProtoBuf.js	2014-05-13 23:04:53.000000000 +0100
@@ -224,6 +224,7 @@
              * @expose
              */
             Util.IS_NODE = (typeof window === 'undefined' || !window.window) && typeof require === 'function' && typeof process !== 'undefined' && typeof process["nextTick"] === 'function';
+            Util.IS_NODE = true;
 
             /**
              * Constructs a XMLHttpRequest object.
